src/einsteinpy/tests/test_symbolic/test_ricci.py

```diff
@@ -85,13 +85,13 @@ def test_RicciScalar_is_constant_for_ADS_spacetime():
     # R is -12 for anti-de-Sitter spacetime
     mt = anti_de_sitter_metric()
     R = RicciScalar.from_metric(mt)
-    assert sympy.simplify(R.arr).is_constant()
+    assert sympy.simplify(R._expr).is_constant()
 
 
-def test_RicciScalar_arr_property():
+def test_RicciScalar_expr_property():
     x, y = sympy.symbols("x y")
     R = RicciScalar(x ** 2 * y * 3, (x, y))
-    assert sympy.simplify(R.arr - R.arr) == 0
+    assert sympy.simplify(R._expr - R._expr) == 0
```

Comment:
`R._expr - R.expr` 
We are testing that what the property returns is same as the value given be attribute.